src/lib/sections/Navigation/Navigation.scss

```diff
@@ -0,0 +1,41 @@
+@import "vanilla-framework";
+@include vanilla;
```

Comment:
This will result int the entire vanilla framework being included in the CSS build (adding ~370kB).

```suggestion
```